procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -112,17 +110,17 @@ mozilla-central->mozilla-beta migration no-op trial run
 mozilla-esr bump no-op trial run
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Follow the `general steps <#general-steps>`__ hopping on `esr <https://treeherder.mozilla.org/#/jobs?repo=mozilla-esr78>`__
+1. Follow the `general steps <#general-steps>`__ hopping on `esr91 <https://treeherder.mozilla.org/#/jobs?repo=mozilla-esr91>`__
```

Comment:
Maybe add a note prior to the instructions to replace `esr91` with whatever the current version is you are bumping

Reply:
Thanks - done.